releases/release-8.4.0.md

```diff
@@ -0,0 +1,230 @@
+
+    Optimize the query performance of cached tables by up to 5.4 times when using `IndexLookup` to execute `SELECT ... LIMIT 1` with `IndexLookup`. Improve the performance of `IndexLookupReader` in full table scan and primary key query scenarios.
+
+### Reliability
+
+* Feature summary [#issue-number](issue-link) @[pr-auorthor-id](author-link)
+
+    Feature descriptions (including what the feature is, why it is valuable for users, and how to use this feature generally)
+
+    For more information, see [documentation](doc-link).
+
+### Availability
+
+* Feature summary [#issue-number](issue-link) @[pr-auorthor-id](author-link)
+
+    Feature descriptions (including what the feature is, why it is valuable for users, and how to use this feature generally)
+
+    For more information, see [documentation](doc-link).
+
+### SQL
+
+* Feature summary [#issue-number](issue-link) @[pr-auorthor-id](author-link)
+
+    Feature descriptions (including what the feature is, why it is valuable for users, and how to use this feature generally)
+
+    For more information, see [documentation](doc-link).
+
+### DB operations
+
+* PITR adds client-side log backup data encryption support (experimental) [55834](https://github.com/pingcap/tidb/issues/55834) @[Tristan1900](https://github.com/Tristan1900) **tw@qiancai** <!--1920-->
+
+    Previously only the data from a snapshot based backup could be encrypted (on the client side) with a data key provided by the user. With this feature, log backups may now also be encrypted, ensuring that the confidentiality of information within the backup data is secured.
+
+    For more information, see [documentation](doc-link).
+
+* BR reduces requires storage permissions for restores [#55870](https://github.com/pingcap/tidb/issues/55870) @[Leavrth](https://github.com/Leavrth) **tw@Oreoxmt** <!--1943-->
+
+    Previously, when BR was restoring data, checkpoint information about the progress of the restore was recorded in the location hosting the backup data. These restore checkpoints enabled restoration to be quickly resumed if it was interrupted. With this feature, the restore checkpoints are now stored in the target TiDB cluster. This means that BR only requires read access to the backup dataset location for restores.
+
+    For more information, see [documentation](doc-link).
+  
+* Feature summary [#issue-number](issue-link) @[pr-auorthor-id](author-link)
+
+    Feature descriptions (including what the feature is, why it is valuable for users, and how to use this feature generally)
+
+    For more information, see [documentation](doc-link).
+
+### Observability
+
+* Feature summary [#issue-number](issue-link) @[pr-auorthor-id](author-link)
+
+    Feature descriptions (including what the feature is, why it is valuable for users, and how to use this feature generally)
+
+    For more information, see [documentation](doc-link).
```

Comment:
```suggestion
* Display the CPU time of TiDB and TiKV in the system table [#55542](https://github.com/pingcap/tidb/issues/55542) @[yibin87](https://github.com/yibin87) **tw@hfxsd** <!--1877-->

      The [Top SQL page](/dashboard/top-sql.md) of [TiDB Dashboard](/dashboard/dashboard-intro.md) displays SQL statements with high CPU consumption. Starting from v8.4.0, TiDB includes CPU time consumption data in the system table, alongside other session or SQL metrics, allowing you to easily monitor high CPU usage from multiple perspectives. This information is especially useful in identifying the root cause of issues such as CPU spikes or hotspots in cluster read/write operations.

    - [STATEMENTS_SUMMARY](/statement-summary-tables.md) adds `AVG_TIDB_CPU_TIME` and `AVG_TIKV_CPU_TIME` to show the average CPU time consumed by individual SQL statements historically.
    - [INFORMATION_SCHEMA.PROCESSLIST](/information-schema/information-schema-processlist.md) adds `TIDB_CPU` and `TIKV_CPU` to display the cumulative CPU consumption of currently executing SQL statements in a session.
    - The [slow Log](/analyze-slow-queries.md) adds the `Tidb_cpu_time` and `Tikv_cpu_time` fields to show the CPU time of captured SQL statements.

  By default, TiKV CPU time is displayed. Collecting TiDB CPU time introduces an additional overhead (about 8%), so TiDB CPU time is only displayed as the actual value when the [Top SQL feature](https://github.com/dashboard/top-sql.md) is enabled; otherwise, it will always display as `0`.
  
    For more information, see [documentation](/information-schema/information-schema-processlist.md) and [documentation](information-schema/information-schema-slow-query.md).
```